x/bank/keeper/send.go

```diff
@@ -266,7 +266,10 @@ func (k BaseSendKeeper) SetBalance(ctx sdk.Context, addr sdk.AccAddress, balance
 	balancesStore := prefix.NewStore(store, types.BalancesPrefix)
 	accountStore := prefix.NewStore(balancesStore, AddressToPrefixKey(addr))
 
-	bz := k.cdc.MustMarshalBinaryBare(&balance)
+	bz, err := balance.Marshal()
+	if err != nil {
+		return err
+	}
```

Comment:
Any reason to change this?
Do we ignore the legacy amino codec?
> https://github.com/line/lbm-sdk/blob/a18b7c8c01a77e5a1290a0a4600feda80ad3abce/codec/codec.go#L15-L16

Reply:
It's done because there's measurable differences in performance between MarshalBinaryBare and protobuf's Marshal.

Yes, we'll ignore legacy codec as these are internal object serialization that's never going to be exposed.